### DISCUSSION TITLE
How to start with older Uno versions?`

### DISCUSSION THREAD

**Discussion by Glacial** [unoplatform/uno#8935] [Q&A]
I'm new to Uno and Xamarin and the like and have tried to follow the getting started guide specifically for trialing the iOS support. The problem I have is that the 2013 macbook I have, frustratingly cannot support Xcode 13.3 (thanks Apple) and I'm pretty sure this is the reason for builds to fail when following the guide. I've tried using VS "Pair to Mac" on Windows and VSFM routes with similar results.

Seeing as Uno has been around a while I was wondering if there is a path to get up and running with older versions of it. There must be a combination of VS and Uno that is happy on Xcode 13.2. If so, can you inform me of what these are and how I can follow some getting started guides from that point in time? I've not been able to find instructions for this myself.

Thanks.

**Comment by jeromelaban** [comment]
Uno can work with older versions of Xamarin (it's required for CI builds), but the problem is with the use of older Visual Studio tooling, unfortunately. You could try installing Visual Studio for mac (prior to the 2022 version), it may support older versions of Xcode.
